allennlp/common/tee_logger.py

```diff
@@ -5,6 +5,16 @@
 from typing import TextIO
 import os
 
+def clean(message: str):
```

Comment:
Can you give this a more descriptive name, like `replace_return_with_newline`?  Also, make the comment into a docstring.